portfolio/src/main/webapp/script.js

```diff
@@ -17,29 +17,69 @@
  * Adds a random quote to the page.
  */
 function addRandomQuote() {
-    const quotes =
-        ['My "people skills" are "rusty"', "What would Hermione do?", "It matters not what someone is born, but what they grow to be.",
-    "And if eating cake is wrong, I don't want to be right."];
-    const authors = 
-        ["Castiel, Supernatural", "Charlie Bradbury, Supernatural", "Albus Dumbledore", "Lorelai Gilmore, Gilmore Girls"];
-    
-        //Pick random quote
-    const randomIndex = Math.floor(Math.random() * quotes.length);
-    const quote = quotes[randomIndex];
-    const author = authors[randomIndex];
-
-    //Add it to the page.
-    const quoteText = document.getElementById('quote-text');
-    quoteText.innerText = '"' + quote + '"';
-
-    const quoteAuthor = document.getElementById('quote-author');
-    quoteAuthor.innerText = "- " + author;
+  const quotes =
+    ['My "people skills" are "rusty"', "What would Hermione do?", "It matters not what someone is born, but what they grow to be.",
+      "And if eating cake is wrong, I don't want to be right."];
+  const authors =
+    ["Castiel, Supernatural", "Charlie Bradbury, Supernatural", "Albus Dumbledore", "Lorelai Gilmore, Gilmore Girls"];
+
+  //Pick random quote
+  const randomIndex = Math.floor(Math.random() * quotes.length);
+  const quote = quotes[randomIndex];
+  const author = authors[randomIndex];
+
+  //Add it to the page.
+  const quoteText = document.getElementById('quote-text');
+  quoteText.innerText = '"' + quote + '"';
+
+  const quoteAuthor = document.getElementById('quote-author');
+  quoteAuthor.innerText = "- " + author;
 }
 
 function getGreeting() {
-    console.log("Fetching the greeting");
+  console.log("Fetching the greeting");
+
+  fetch('/data').then(response => response.text()).then((greeting) => {
+    document.getElementById('greeting-container').innerText = greeting;
+  });
+}
+
+// function getMessages() {
```

Comment:
In generally, large blocks of commented out code shouldn't be left in commits.

Reply:
Oops, forgot about that :)